amundsen_application/api/preview/dashboard/v0.py

```diff
@@ -45,6 +45,7 @@ def get_preview_image(uri: str) -> Response:
                          mimetype='image/jpeg',
                          cache_timeout=app.config['DASHBOARD_PREVIEW_IMAGE_CACHE_MAX_AGE_SECONDS'])
```

Comment:
File a ticket to add proper docs about the dashboard preview in the same way that we have docs for table preview.